.github/workflows/update-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Update extension test
+on:
+  push:
+    branches-ignore:
+      - master
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: styfle/cancel-workflow-action@0.11.0
+      - run: sudo chown -R $USER:$USER $GITHUB_WORKSPACE
+      # Build current branch
+      - name: Get Current branch source
+        uses: actions/checkout@v3
+        with:
+          clean: false
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version-file: .nvmrc
+      - run: yarn install --frozen-lockfile
+      - run: yarn run build
+      - name: Copy current dist to temp folder
+        run: |
+          sudo mkdir -m 777 new_dist
+          mv dist/chrome new_dist
```

Comment:
`sudo` seems unnecessary here. As well as `mkdir`.

Why not just move the whole dir like `mv dist dist.new`?